iree/tools/compilation.bzl

```diff
@@ -15,11 +15,31 @@ def iree_bytecode_module(
         flags = ["-iree-mlir-to-vm-bytecode-module"],
         translate_tool = "//iree/tools:iree-translate",
         embedded_linker_tool = "@llvm-project//lld:lld",
+        opt_tool = "//iree/tools:iree-opt",
+        opt_flags = [],
```

Comment:
Hrm I think iree-translate should be able to do everything here. Generally it has all the passes. Now there's some question of whether it *should* be able to do that, but I think if your pass flags here were before the other flags then they would get run first. Can you try just overriding flags?